CONTRIBUTING.md

```diff
@@ -78,25 +74,25 @@ All documentation must abide by the following maxims:
 - Generation should be **completely automated** and available for scripting.
 - Artifacts that must be included in the Typesafe stack should be published to a maven “documentation” repository as documentation artifacts.
 
-All documentation is preferred to be in Typesafe's standard documentation format [reStructuredText](http://doc.akka.io/docs/akka/snapshot/dev/documentation.html) compiled using Typesafe's customized [Sphinx](http://sphinx.pocoo.org/) based documentation generation system, which among other things allows all code in the documentation to be externalized into compiled files and imported into the documentation.
+All documentation is preferred to be in Typesafe's standard documentation format [reStructuredText](https://docutils.sourceforge.io/rst.html) compiled using Typesafe's customized [Sphinx](https://www.sphinx-doc.org/en/master/) based documentation generation system, which among other things allows all code in the documentation to be externalized into compiled files and imported into the documentation.
```

Comment:
I don't find any document files in this project, is this statement still applied?

Reply:
Good point. I have replaced that section with a new one that better describes the current situation.

At some stage in the future, we should add docs that are published to the Pekko web site but I don't want to block a 1.0.0 release over this.